packages/editor-ui/src/components/ParameterInput.vue

```diff
@@ -649,7 +645,18 @@ async function loadRemoteParameterOptions() {
 			credentials: node.value.credentials,
 		});
 
-		remoteParameterOptions.value = remoteParameterOptions.value.concat(options);
+		const [safeOptions, invalidOptions] = partition(options, isINodePropertyOptions);
+
+		if (invalidOptions.length > 0) {
+			captureMessage('Invalid parameter options', {
+				extra: {
```

Comment:
Let's use `error` level since this is an error

```suggestion
			captureMessage('Invalid parameter options', {
				level: 'error',
				extra: {
```